lighthouse-core/report/html/report-styles.css

```diff
@@ -434,6 +433,14 @@
   border-bottom: var(--score-icon-size) solid var(--color-fail);
 }
 
+.lh-audit--error .lh-audit__score-icon {
+  background-image: var(--error-icon-url);
+  width: calc(var(--score-icon-size) * 2);
+  height: calc(var(--score-icon-size) * 2);
+  transform: translateX(-6px);
+  margin-right: 0;
+}
```

Comment:
2x does seem too big. 1.7, 1.75?

Also, what do you think about using scaling instead of width/height to avoid the margin shenanigans. Like
```css
.lh-audit--error .lh-audit__score-icon {
  background-image: var(--error-icon-url);
  transform: scale(1.7);
}
```

Reply:
@brendankenny is it possible to share a screenshot of the current icon size?

Reply:
### 1x
<img width="456" alt="1x error icon" src="https://user-images.githubusercontent.com/316891/60300755-83a36600-98e4-11e9-995e-ffe29506cccd.png">

<hr>

### 1.7x:
<img width="461" alt="1.7x error icon" src="https://user-images.githubusercontent.com/316891/60300771-9453dc00-98e4-11e9-9572-93fe41d4eb09.png">

<hr>

### 2x:
<img width="461" alt="2x error icon" src="https://user-images.githubusercontent.com/316891/60300838-b5b4c800-98e4-11e9-838d-7daf892a6600.png">

Reply:
one issue is that the scaling doesn't do great things to rendering of the `!`. It gets pretty muddled at less than a ~1.7x scale

Reply:
Looking at this mock, i have more context for what you are trying to do. 
The **original Red triangle** already signifies "Error", and doesn't need to be changed. Replacing it with the Material icon feels out of place and creates more confusion for users. 

How about the most simple & literal approach like this: 
<img width="431" alt="Screen Shot 2019-06-27 at 3 33 13 PM" src="https://user-images.githubusercontent.com/406636/60305273-e995ea80-98f0-11e9-8ade-2a44da9d1566.png">

Reply:
> The original Red triangle already signifies "Error", and doesn't need to be changed. Replacing it with the Material icon feels out of place and creates more confusion for users.

But today it's not an error, it signifies the page did poorly at something. A really bad FCP is very different than "something went wrong and LH has no idea what your FCP was".

One example of the confusion that sometimes pops up is in #8287

Reply:
@brendankenny my  understanding of **Runtime Error** is "something went wrong and LH doesn't know how to properly produce audit result/score". Maybe we should come up with a more clear label w/o having the word **error** in it? Something like "Audit invalid", or "Audit unavailable", or "Invalid result" ?  

As for the big score with ? mark, might be good to have a info tooltip briefly explain why the score is unavailable.